lib/timescale/compression_settings.rb

```diff
@@ -1,6 +1,7 @@
 module Timescale
-  class CompressionSettings < ActiveRecord::Base
+  class CompressionSetting < ActiveRecord::Base
     self.table_name = "timescaledb_information.compression_settings"
     belongs_to :hypertable, foreign_key: :hypertable_name
   end
+  CompressionSettings = CompressionSetting
```

Comment:
Maybe a CHANGELOG would be the right place to document this? Ideally we'd just delete those lines, but that would break any pre-existing external code that used those constants. The gem hasn't hit 1.0 yet, so that's probably not a huge deal. Probably.